cli/src/cli_util.rs

```diff
@@ -260,10 +260,23 @@ impl TracingSubscription {
     pub fn enable_debug_logging(&self) -> Result<(), CommandError> {
         self.reload_log_filter
             .modify(|filter| {
+                // the default is INFO
+                // jj-lib and jj-cli are whitelisted for DEBUG logging
+                // this ensures that other crates logging doesn't come int by default
```

Comment:
nits:
* Start lines 1 and 3 with uppercase. End all lines with period.
* crates => crates'
* int => into play (?), or rephrase it